src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -182,12 +183,13 @@ class DraftAdjudicationService(
 
 fun DraftAdjudication.toDto(): DraftAdjudicationDto = DraftAdjudicationDto(
   id = this.id!!,
-  adjudicationNumber = this.reportNumber,
   prisonerNumber = this.prisonerNumber,
   createdByUserId = this.createdByUserId,
   createdDateTime = this.createDateTime,
   incidentStatement = this.incidentStatement?.toDo(),
   incidentDetails = this.incidentDetails.toDto(),
+  adjudicationNumber = this.reportNumber,
+  startedByUserId = this.reportNumber?.let { this.reportByUserId } ?: this.createdByUserId,
```

Comment:
This is where the magic happens.

I had to come up with a name that wasn't `createdByUserId` or `reportByUserId`, but indicated that it was the person who created and reported it (it will always be the same person)!

Reply:
I might be confused here - but couldn't this always be reportedByUserId (is this not set on the Draft?) and when we create a new one for reporting purposes it is also set at that point?

Reply:
Yes - you are confused! 

When we create a draft from scratch, only the `createdByUserId` is set (via Spring's auditing).

Only when we created a draft from an n existing/reported/submitted adjudication do we populate the `reportByUserId `

Reply:
Maybe using the `createdByUserId`in the first place is causing too much confusion, and we ought to just have a `reportedByUserId` field in the draft from the start.

..but that would be a separate PR